From da5192b624d7953b212f62b874cb752974c8a354 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 21 Jul 2021 21:45:14 +0200
Subject: [PATCH] python-ubus: update to version 0.1.2

Version 0.1.2 includes change for ubus socket due to upstream change
since OpenWrt 21.02
---
 lang/python-ubus/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lang/python-ubus/Makefile b/lang/python-ubus/Makefile
index ddd1cbdee..1af372bce 100644
--- a/lang/python-ubus/Makefile
+++ b/lang/python-ubus/Makefile
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-ubus
-PKG_VERSION:=0.1
-PKG_RELEASE:=2
+PKG_VERSION:=0.1.2
+PKG_RELEASE:=1
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=LGPLv2.1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/python-ubus.git
-PKG_MIRROR_HASH:=8a1ee1eaa811d983ccdef186f3f248a099867e651575f56b72318367f19d56e9
+PKG_MIRROR_HASH:=ab8fb7ca84ee22fa4f3ca96c3c130e619e346f9abad4c1da5c72cb951acb043b
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 
-- 
2.30.2

